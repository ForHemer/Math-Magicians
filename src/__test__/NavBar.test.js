import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../components/NavBar';

test('renders correctly', () => {
  const nav = renderer.create(<Router><NavBar /></Router>).toJSON();
  expect(nav).toMatchSnapshot();
});
