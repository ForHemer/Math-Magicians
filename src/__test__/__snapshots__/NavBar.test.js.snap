// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav>
  <div>
    <a
      className="logo"
      href="/math-magicians"
      onClick={[Function]}
    >
      Math Magicians
    </a>
  </div>
  <ul>
    <li>
      <a
        className=""
        href="/math-magicians"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className=""
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
